src/brogue/Items.c

```diff
@@ -6033,6 +6048,52 @@ void relabel(item *theItem) {
     }
 }
 
+// If the most recently equipped item caused another item to be unequiped, is
+// uncursed, and both haven't left the inventory since, swap them back.
+void swap() {
```

Comment:
```suggestion
void swapLastEquipment() {
```
Or something

Reply:
I had almost done something like that, but the other methods are named rather literally after their action labels in the help.  I'll change it.

Reply:
I'm also open to other names for this action, I ended up on "sWap" because it is short and `w` was open.

If there is another name and suitable lower-case key on the left side of the board, I'd be game.  I would rather avoid pressing shift; coordinating two key presses is almost as back as having to press `i` first.

Maybe `q` and "reeQuip"? or "Quick swap"?

Reply:
I think `w` works